app/src/main/res/layout/insurance_contract_card.xml

```diff
@@ -69,12 +69,13 @@
                     tools:visibility="visible" />
 
                 <ImageView
+                    android:id="@+id/logo"
                     android:layout_width="@dimen/contract_card_logo_size"
                     android:layout_height="@dimen/contract_card_logo_size"
                     android:contentDescription="@null"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintTop_toTopOf="parent"
-                    app:srcCompat="@drawable/ic_hedvig_h"
+                    tools:src="@drawable/ic_hedvig_h"
```

Comment:
Is there any specific reason for this change? I don't know from the top of my head what the difference is, but usually "compat = good" 😂

Reply:
The `ImageView` no longer loads an image resource from disk, but instead gets its image from `coil`. The switch to `tools:src` was merely to make sure something is displayed in the preview window. 🙂

Reply:
Oh "tools"! Missed the most important change here duh